packages/api-sdk/src/resources/user.ts

```diff
@@ -1,24 +1,20 @@
-import atob from 'atob';
+import JWT from 'jsonwebtoken';
 
 import { CreatorID } from '@/models';
-import { getWindow } from '@/utils';
 
 export const parseJWT = <S>(token: string): S => {
-  const base64Url = token.split('.')[1];
```

Comment:
so here in the original, we are completely ignoring the `header` portion.

Reply:
we should probably make these `.split()` calls more verbose to make it clearer what's going on

ex:

```js
const [_header, base64Url] = token.split('.');
```